app/src/main/res/layout/fragment_list_dashboards.xml

```diff
@@ -4,10 +4,68 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent">
 
-    <androidx.recyclerview.widget.RecyclerView
-        android:id="@+id/dashboard_list"
+    <LinearLayout
+        android:id="@+id/progress_bar_container"
         android:layout_width="match_parent"
-        android:layout_height="match_parent" />
+        android:layout_height="match_parent"
+        android:gravity="center">
+
+        <ProgressBar
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content" />
+
+    </LinearLayout>
+
+    <ViewAnimator
+        android:id="@+id/view_animator"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent">
+
+        <LinearLayout
+            android:id="@+id/dashboard_list_container"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:paddingStart="10dp"
+            android:paddingTop="10dp"
+            android:paddingEnd="10dp"
```

Comment:
You can use `android:paddingHorizontal="10dp"` instead of  `android:paddingStart="10dp"`  and `android:paddingEnd="10dp"` 😉

Reply:
Yeah! you also have the option of using `paddingVertical` you can check [here](https://developer.android.com/about/versions/oreo/android-8.0?hl=es-419#ulmp)